static/js/buddy_list.js

```diff
@@ -265,6 +265,9 @@ class BuddyList extends BuddyListConf {
         // Add a fudge factor.
         height += 10;
 
+        // Remove tooltips.
+        $(".tooltip").remove();
```

Comment:
Can we scope this selector to just tooltips over the buddy list area?  Otherwise, we risk background rerendering of this component disappearing a tooltip somewhere else on the page.

(I'm also not sure this is the right/complete fix -- shouldn't we be removing the other tooltips when the hover ends, rather than waiting until the next rerender?)  I can imagine this implementation resulting in it feeling like the right sidebar tooltips randomly disappear if a lot of users are joining and organization in quick succession.

Reply:
Thanks for the feedback!
About your second point, I presume we would use a mouseleave event to detect when a hover ends. There are certain situations where this doesn't seem to work, however. For example, when searching for users in the buddy list, mouseleave is not triggered and cannot be used to remove lingering tooltips. This is why I thought it's best to just clear everything each time the list renders.